.azure-devops/templates/validate-refdoc-generation.yml

```diff
@@ -28,5 +28,5 @@ steps:
         WHEELS=$(ls $ARTIFACTS_DIR/azure-iot/*.whl)
         az --version
         for i in $WHEELS; do
-          python ../azure-cli-extensions/scripts/refdoc/generate.py -e $i
+          python ../azure-cli-extensions/scripts/refdoc/generate.py -e $i -d None
```

Comment:
Good catch on this one - it looks like a fix went in for this issue here, so we might be ok without this workaround:
https://github.com/Azure/azure-cli-extensions/pull/6521